README.md

```diff
@@ -32,27 +34,27 @@ Core Server (i.e. SERVER) project are **public**.
 ## Installation
 `MongoSwift` works with Swift 4.0+.
 
-### FIRST: Install the MongoDB C Driver
-Because the driver wraps the MongoDB C driver, using it requires having the C driver's two components, `libbson` and `libmongoc`, installed on your system. The minimum required version of the C Driver is **1.13.0**.
+### OS X and Linux
+
+Installation on OS X and Linux is supported via [Swift Package Manager](https://swift.org/package-manager/).
+
+#### Step 1: Install the MongoDB C Driver
+The driver wraps the MongoDB C driver, and using it requires having the C driver's two components, `libbson` and `libmongoc`, installed on your system. The minimum required version of the C Driver is **1.13.0**.
 
 On a Mac, you can install both components at once using [Homebrew](https://brew.sh/):
 `brew install mongo-c-driver`.
 
-Or on Linux, use `apt-get` to install `libmongoc` (which includes `libbson` as a dependency) and `pkg-config` (which enables Swift Package Manager to find the components):
+On Linux, use `apt-get` to install `libmongoc` (which includes `libbson` as a dependency) and `pkg-config` (which enables Swift Package Manager to find the components):
 ```
 sudo apt-get install pkg-config
 sudo apt-get install libmongoc-1.0.0
 ```
 
-Alternatively, see the [installation guide](http://mongoc.org/libmongoc/current/installing.html) from libmongoc's documentation.
+See the [installation guide](http://mongoc.org/libmongoc/current/installing.html) from libmongoc's documentation for more information.
```

Comment:
Should this be `` `libmongoc` `` like in its other mentions? ([example](https://github.com/mongodb/mongo-swift-driver/pull/147/files#diff-04c6e90faac2675aa89e2176d2eec7d8R47))

Reply:
done.